app/frontend/src/components/autocomplete/autocomplete.js

```diff
@@ -1,37 +1,46 @@
-import 'es6-promise/auto';
-import view from './autocomplete.view';
+import accessibleAutocomplete from 'accessible-autocomplete';
+import 'accessible-autocomplete/dist/accessible-autocomplete.min.css';
+import './autocomplete.scss';
 
-export const renderAutocomplete = (widgetParams) => {
-  view.create(widgetParams.container, widgetParams.input, widgetParams.key);
+const AUTOCOMPLETE_THRESHOLD = 3;
 
-  widgetParams.input.addEventListener('input', () => {
-    if (isActive(widgetParams.threshold, widgetParams.input.value.length)) {
-      autocomplete.showOptions(widgetParams.getOptions, widgetParams.container, widgetParams.input, widgetParams.key);
-    } else {
-      autocomplete.clearOptions(widgetParams.container, widgetParams.input);
-    }
-  });
-};
-
-export const removeUnwantedCharacters = (searchTerm) => searchTerm.replace('.', ' ');
+const highlightRefinement = (text, refinement) => {
+  const index = text.toLowerCase().indexOf(refinement.toLowerCase());
 
-export const showOptions = (getOptions, container, input, key) => {
-  const searchTerm = removeUnwantedCharacters(input.value);
-  return getOptions(searchTerm).then((options) => view.show(options, container, input, key));
-};
+  if (index >= 0) {
```

Comment:
Arguably you don't need this condition, right? If index is 0 we would get `text.substring(0, 0)` and that would (correctly) highlight nothing.